temporal-sdk/src/main/java/io/temporal/internal/statemachines/WorkflowStateMachines.java

```diff
@@ -135,7 +133,7 @@
   private final Map<String, MutableSideEffectStateMachine> mutableSideEffects = new HashMap<>();
 
   /** Key is changeId */
-  private final Map<String, VersionStateMachine> vesions = new HashMap<>();
+  private final Map<String, VersionStateMachine> versions = new HashMap<>();
```

Comment:
Thank you lol :pray: this has bugged me reading the code but I hadn't fixed it yet